src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,17 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Abstract class containing 2 methods for the command objects to implement
+ */
+public abstract class Command {
+
+    public abstract String execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
+
+
+    public abstract boolean isExit();
```

Comment:
Perhaps could include javadocs for these 2 methods in Command class and omit in child classes. Then, you could use @inheritDoc if the method in child has slightly different behavior from parent method.